### ISSUE TITLE
Current time is smaller than the first trajectory point produced by the Planning module.

### ISSUE THREAD

**Issue by LIIHWF** [ApolloAuto/apollo#14855]
### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**: 18.04
- **Apollo installed from (source or binary)**: source
- **Apollo version (3.5, 5.0, 5.5, 6.0)**: 7.0

### Steps to reproduce the issue:
I'm using MODE_MOCK to simulate Apollo Autopilot, and found that the first trajectory produced by the Planning module can be larger than the current time. Here is how I triggered this problem:
- Update perception information for every 0.05 seconds.
- The Planning module will be triggered to execute once per 0.05 seconds.
- However, only the first produced trajectory contains a point with relative_time=0. For subsequent trajectories, the relative_time of the first point is 0.1 seconds, which exceeds the current time of 0.05 seconds.

Is this the correct result? Do I need to configure any parameters to improve this?

**Comment by YuqiHuai** [comment]
Maybe the issue you described is causing SimControl to produce weird scenarios ...

https://user-images.githubusercontent.com/34195403/228130073-e2a86225-22c9-401f-a4b5-9c9e8abf84e2.mp4

**Comment by starrye** [comment]
> @LIIHWF Based on
> 
> https://github.com/ApolloAuto/apollo/blob/b00c030ac02c54f98579d3d3e879dbf52f0d1d53/modules/dreamview/backend/sim_control_manager/dynamic_model/perfect_control/sim_perfect_control.h#L167
> 
> I would guess the developers are expecting the Prediction frequency to be 10 Hz, and since Planning is triggered by Prediction then Planning should also be producing messages at 10 Hz. I haven't looked into the details of trajectory points' time intervals and I am also hoping someone more knowledgeable about this can determine what is causing the issue and why SimControl is shown to be teleporting back and forth.
> 
> I also guess this wouldn't have a real-world effect. Because Apollo sends control commands to LG SVL and I haven't seen anything weird about ego car movement in SVL-based scenarios besides the shaking issue that might have been caused by delays.

apollo的planning日志有什么异常的输出吗？比如一直replan什么的

**Comment by LIIHWF** [comment]
> apollo的planning日志有什么异常的输出吗？比如一直replan什么的

Yes. In my case, the planning module reports warnings in the log:
```
[trajectory_stitcher.cc:156] current time smaller than the previous trajectory's first time
```

The parameter `is_replan` in the output is always `true`.

**Comment by YuqiHuai** [comment]
@starrye @LIIHWF Same here. I also noticed planning keeps sending "rerouting requests".
